### ISSUE TITLE
[GitHub Support] pattern filter to watching specific branch, such as master, dev-*(wildcard)

### ISSUE THREAD

**Issue by takahi-i** [walter-cd/walter#105]

**Comment by takahi-i** [comment]
I will look into this issue.

**Comment by takahi-i** [comment]
```
service:
  type: github
  token: ADD_YOUR_KEY
  repo: YOUR_REPOSITORY_NAME
  from: YOUR_ACCOUNT_OR_GROUP_NAME
  update: UPDATE_FILE_NAME
  pattern: dev-*
````

**Comment by takahi-i** [comment]
Fixed with #113
